packages/assistant/src/RPA/Assistant/library.py

```diff
@@ -500,6 +502,12 @@ def add_text_input(
         to add a descriptive label and the ``placholder`` text can be given
         to act as an example of the input value.
 
+        The `default` value will be assigned to the input field if the user
+        doesn't complete it. If provided, the placeholder won't be shown.
+        By default it will be an `None`. Also, if a default value is provided
```

Comment:
```suggestion
        This is `None` by default. Also, if a default value is provided
```

Reply:
thanks! writing docs is hard 😅  changed to your suggestion